src/option.ts

```diff
@@ -1,15 +1,30 @@
 /**
  * @since 0.5.0
  */
-import * as t from 'io-ts'
 import { Option } from 'fp-ts/lib/Option'
+import * as t from 'io-ts'
 
 const None = t.strict({
   _tag: t.literal('None')
 })
 
 const someLiteral = t.literal('Some')
 
+/**
+ * @since 0.5.18
+ */
+export type NoneOutput = t.OutputOf<typeof None>
```

Comment:
is there a particular reason for using `t.OutputOf` here? 

I was expecting `export type NoneOutput = { _tag: 'None' }`

Reply:
Preventing double definition and possible divergence.